modules/file-integrity-understanding-file-integrity-cr.adoc

```diff
@@ -23,10 +23,34 @@ metadata:
   name: worker-fileintegrity
   namespace: openshift-file-integrity
 spec:
-  nodeSelector:
+  nodeSelector: <1>
       node-role.kubernetes.io/worker: ""
-  config: {}
+  tolerations: <2>
+  - key: "myNode"
+    operator: "Exists"
+    effect: "NoSchedule"
+  config: <3>
+    name: "myconfig"
+    namespace: "openshift-file-integrity"
+    key: "config"
+    gracePeriod: 20 <4>
+    maxBackups: 5 <5>
+    initialDelay: 60 <6>
+  debug: false
+status:
+  phase: Active <7>
 ----
+<1> Defines the selector for scheduling node scans.
+<2> Specify `tolerations` to schedule on nodes with custom taints. When not specified, a default toleration allowing running on master and infra nodes is applied.
```

Comment:
is this one of those cases where master is allowed? Or can we use control plane?

Reply:
Good catch, I'm still bad at that. 

I think we can use "main" in this instance.